src/Jaeger/Tracer.php

```diff
@@ -203,10 +203,8 @@ public function startSpan($operationName, $options = [])
             $options->getStartTime()
         );
 
-        if (($rpcServer || $parentId === null) && ($flags & SAMPLED_FLAG)) {
-            // this is a first-in-process span, and is sampled
-            $span->setTags($this->tags);
-        }
```

Comment:
Why do you changed this?

Reply:
Hi, the `hostname`, `jaeger_version` and other tags that are present on the tracer are not attached on the spans if the span is child_of other span. In my case I receive a trace-id from another service and start my spans from it. All my services are load balanced so there is no way to know which instance returns me the results.

By removing that and replacing it with:
```php
$mergedTags = array_merge($this->tags, $tags);
``` 
I am sure that the tracer spans are always present, and in case someone needs to overwrite something on a specific span they can.